core/src/apps/ethereum/clear_signing.py

```
@@ -1172,26 +1172,78 @@ async def _expand_calldata_field(
+            field_definition, formatter, blobs, callees, msg, defs, nested
+        )
+
+    if isinstance(callees, list):
+        if len(callees) != len(blobs):
+            raise InvalidFormatDefinition
+    else:
+        # Same callee for all subcalls
+        callees = [callees] * len(blobs)
+
+    rows: list[DisplayedField] = []
+    for i, (blob, callee) in enumerate(zip(blobs, callees)):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)